app/views/shared/_sponsors.html.erb

```diff
@@ -0,0 +1,54 @@
+<section class="container">
+
+  <h1>
+    Proudly sponsored by
+  </h1>
+
+  <div class="col-md-4">
+    <div class="card-block">
+      <img src="https://res.cloudinary.com/dnbyfobad/image/upload/v1490716457/event1_iweuk3.jpg" height=50/>
+      <p class="card-text">
+        <p>Sponsorname</p>
+      </div>
+    </div>
+
+    <div class="col-md-4">
+      <div class="card-block">
+        <img src="https://res.cloudinary.com/dnbyfobad/image/upload/v1490716457/event1_iweuk3.jpg" height=50/>
+        <p class="card-text">
+          <p>Sponsorname</p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="card-block">
+          <img src="https://res.cloudinary.com/dnbyfobad/image/upload/v1490716457/event1_iweuk3.jpg" height=50/>
+          <p class="card-text">
+            <p>Sponsorname</p>
+          </div>
+        </div>
+
+        <div class="col-md-4">
+          <div class="card-block">
+            <img src="https://res.cloudinary.com/dnbyfobad/image/upload/v1490716457/event1_iweuk3.jpg" height=50/>
+            <p class="card-text">
+              <p>Sponsorname</p>
+            </div>
+          </div>
+
+          <div class="col-md-4">
+            <div class="card-block">
+              <img src="https://res.cloudinary.com/dnbyfobad/image/upload/v1490716457/event1_iweuk3.jpg" height=50/>
+              <p class="card-text">
+                <p>Sponsorname</p>
+              </div>
+            </div>
+
+            <div class="col-md-4">
+              <div class="card-block">
+                <img src="https://res.cloudinary.com/dnbyfobad/image/upload/v1490716457/event1_iweuk3.jpg" height=50/>
+                <p class="card-text">
+                  <p>Sponsorname</p>
+                </div>
+              </div>
+            </section>
```

Comment:
fix indentation / close all your elements - cant see what is the problem here :/